### ISSUE TITLE
ELK Batman introspection to support multiple URLs of the ElasticSearch

### ISSUE THREAD

**Issue by ateska** [TeskaLabs/seacat-auth#302]
That's because the ElasticSearch typically runs in the cluster.

It is important for the automated deployment as well.

**Comment by eliska-n** [comment]
beruška!
Could this use the common elasticsearch config section?
```
[elasticsearch]
url = http://elastic-1:9200/
      http://elastic-2:9200/
username = <username>
password = <password>
```

**Comment by ateska** [comment]
Yes, please ... it could be a batman ELK fallback if no value is provided in the section.

**Comment by byewokko** [comment]
resolved by #326
